lib/src/util/class_names.dart

```diff
@@ -87,10 +88,12 @@ class ClassNameBuilder {
       ..blacklist(cssClassProps.classNameBlacklist);
   }
 
-  /// Adds a className string. May be a single CSS class 'token', or multiple space-delimited classes,
-  /// IF [should] is true, otherwise, does nothing (convenience for helping to inline addition conditionals).
+  /// Adds all of the CSS classes represented by [className] _(a space-delimited list)_ to the [_classNamesBuffer] -
```

Comment:
`to the [_classNamesBuffer] ` is an irrelevant (and possibly confusing) implementation detail (and can be seen via the dartdoc or by clicking into the source, if desired).